### PR TITLE
Added Operator Tips

### PR DESCRIPTION
**Pull request by nerdydread**
I added some tips for working with Tyk Operator in light of a recent issue encountered during a Tyk Operator demo.  See slack thread [here](https://tyktech.slack.com/archives/C01ABR6JLMQ/p1660600044984959) for reference.

### CODE REVIEW

**Review by marksou — APPROVED** [review]

File: tyk-docs/content/tyk-stack/tyk-operator/installing-tyk-operator.md [hunk 1]
```diff
@@ -250,4 +255,11 @@ If you think we did something wrong, you can create a [GitHub issue](https://git
 ```bash
 helm delete tyk-operator -n tyk-operator-system
 ```
+
+### Troubleshooting Tyk Operator
+If you experience issues with the behavior of the Tyk Operator (e.g. API changes not being applied), to investigate, you can check the logs of the tyk-operator-controller-manager pod in your cluster with the following command:
+
+```bash
+kubectl logs <tyk-controller-manager-pod-name> -n tyk-operator-system
```
Comment (buraksekili) [inline tyk-docs/content/tyk-stack/tyk-operator/installing-tyk-operator.md]:
```suggestion
kubectl logs <tyk-controller-manager-pod-name> -n tyk-operator-system manager
```

**Comment by carolwkc** [comment]
Thanks @nerdydread . LGTM with @buraksekili 's suggestion.

**Review by carolwkc — APPROVED** [review]
Looks good, thanks!

**Pull request merged**